torchdata/datapipes/iter/transform/flatmap.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+from torch.utils.data import DataChunk
+from torch.utils.data import functional_datapipe, IterDataPipe
+from torch.utils.data.datapipes.utils.common import DILL_AVAILABLE, check_lambda_fn
+
+from typing import TypeVar
+
+if DILL_AVAILABLE:
+    import dill
```

Comment:
The Error from `mypy` can be ignored here. I have another PR that should solve it.